tests/framework/db/pgsql/CommandTest.php

```
@@ -149,6 +154,183 @@ public function testAutoQuoting(): void
+                'item',
+                ['name' => 'invalid category while enabled', 'category_id' => -2],
+            )->execute();
+        } finally {
+            $command->checkIntegrity(true, '', 'item')->execute();
+            $pdo->setAttribute(PDO::ATTR_EMULATE_PREPARES, $originalEmulatePrepares);
+        }
+    }
+
+    public function testCheckIntegrityPreservesEmulatedPrepares(): void
+    {
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)